build/minio.sh

```diff
@@ -37,6 +38,7 @@ install_minio ()
     --set environment.MINIO_SSE_AUTO_ENCRYPTION=on \
     --set environment.MINIO_SSE_MASTER_KEY=my-minio-key:feb5bb6c5cf851e21dbc0376ca81012a9edc4ca0ceeb9df5064ccba2991ae9de \
     --set accessKey="AKIAIOSFODNN7EXAMPLE",secretKey="wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
+    --set resources.requests.memory=2Gi,persistence.size=${pvc_size} \
```

Comment:
Do we really need 2Gi memory requests? Can you check if 100Mi/200Mi would be enough?

Reply:
we don't even have to change the persistence size. That was not the problem. 
We were just supposed to handle the resource that minio needs. So memory basically.

Reply:
added pvc change because of [this](https://github.com/kanisterio/kanister/issues/1176#issuecomment-1122817635) comment.

Reply:
reverted pvc change